### ISSUE TITLE
V1 Release ⭐

### ISSUE THREAD

**Issue by brillout** [vikejs/vike#985]
Once the https://github.com/vikejs/vike/labels/v1%20release%20%3Astar2%3A tickets are implemented, Vike can be considered a feature complete, highly polished, and robust tool to build modern apps.

Plan:

- [x] Rename vite-plugin-ssr to Vike.
- [x] Stable release of the https://github.com/brillout/vite-plugin-ssr/issues/578 (and soft-deprecate old design).
- [ ] Implement https://github.com/vikejs/vike/labels/v1%20release%20%3Astar2%3A tickets.
- [x] Rehaul marketing pages to target a wider audience with a more professional/dry tone. (The current copy has more of a personal/hobbyist/passionate tone.) Also make `vike.dev` prettier.
- [ ] Marketing, marketing, and marketing. (Lots of comments and blog posts on social news sites, reach out to news sites & influencers, automatically track relevant news, ...)

**Comment by BlackCubeNo99** [comment]
Is v1 okay?

**Comment by brillout** [comment]
@BlackCubeNo99 Yes the v1 roadmap is still the plan. (Althoug the Vike rename may happen earlier than planned.)

**Comment by harrytran998** [comment]
I just commented here to see the day Vike released v1 - To see your sweet of your efforts in a long time!
Thank you!

**Comment by redbar0n** [comment]
> Rehaul marketing pages to target a wider audience with a more professional/dry tone. (The current copy has more of a personal/hobbyist/passionate tone.)

Please don’t. :) Professional/dry tone is very uninspiring, and the marketing pages are very detailed now, just like developers like it.

**Comment by brillout** [comment]
I do intend to keep a unique touch to Vike's copy (e.g. I'll probably keep the part about us being passionate), but I do think that a professional/dry tone is paramount for gaining trust: we're surious about what we do and I want to communicate that.

**Comment by brillout** [comment]
ETA: end of this summer.

**Comment by nestle49** [comment]
> ~ETA: end of this summer.~
> 
> ETA: 2025 Q1

Q4 now. still 2025?

**Comment by brillout** [comment]
Current plan is 2026.

@nestle49 Do you need 1.0 ?

The overall API is already "1.0" — it will just be a handful of minor breaking changes (nothing substantial).

**Comment by brillout** [comment]
ETA: 2027. See updated plan in OP.

(We're currently tinkering and building AI tooling. Beyond web frameworks — which are here to stay — AI is obviously the hot topic right now. Not only AI itself, but also AI *tooling* is going to become a central part of web development — hand in hand with web frameworks.)
